assets/runtime/functions

```diff
@@ -226,6 +238,9 @@ redmine_configure_database() {
 
   if [[ ${DB_ADAPTER} == postgresql ]]; then
     exec_as_redmine sed -i "/reconnect: /d" ${REDMINE_DATABASE_CONFIG}
+  elif [[ ${DB_ADAPTER} == "sqlite3" ]]; then
+    exec_as_redmine sed -i "/reconnect: /d" ${REDMINE_DATABASE_CONFIG}
+    exec_as_redmine sed -i "/pool: /d" ${REDMINE_DATABASE_CONFIG}
```

Comment:
I'm not sure if "pool:" does anything but it shows up in several examples and didn't seem to break anything... I didn't see any reason to delete it.